### PR TITLE
Scrollable county tables 

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #1884 

[CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/table-scroll)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/table-scroll/)

Changes proposed in this pull request:
- makes county tables shorter, scrollable
- automatically brings selected county to top of chart on clikc
- county charts it works in:
  - [x] state federal production
  - [x] state federal revenue
  - [x] state jobs
  - [x] offshore federal production
  - [x] offshore federal revenue

/cc @ericronne

### CODE REVIEW

**Comment by ericronne** [comment]
Everything seems cool except for offshore revenue tables, which are misbehaving :/

File: _sass/blocks/_county-bar-table.scss [hunk 1]
```diff
@@ -50,4 +50,20 @@
       background-color: $green-dark;
     }
   }
+
+  .inner-table-cell {
+    padding: 0;
+  }
+
+  .inner-table-wrapper {
+    max-height: 12em;
```
Comment (gemfarmer) [inline _sass/blocks/_county-bar-table.scss]: fixed.

**Comment by gemfarmer** [comment]
@ericronne I determined that that issue actually is related to this PR. Fixing now...

**Comment by gemfarmer** [comment]
@ericronne ready for merge as of cfc0d52

**Comment by ericronne** [comment]
💃 💥

**Pull request merged**